editor/create_dialog.h

```diff
@@ -54,11 +54,13 @@ class CreateDialog : public ConfirmationDialog {
 	LineEdit *search_box;
 	Tree *search_options;
 	bool is_replace_mode;
+	Node *node_to_replace;
```

Comment:
You can convert this field to a local variable.
Note that it is only used in the call to `StringName Object::get_class_name()`, so it can probably be converted to `Object*` or even directly to `StringName`.

Reply:
Thank you for these comments! I'm already fixing some of these formatting mistakes, and I didn't know about clang-format. Will definitely remember that. 

This one was makes a lot of sense. It's unnecessary for me to point to a full node when the only thing being accessed is the name. I'll do this right now, run that clang-format, and re commit my changes.

Reply:
Make sure to [rebase](http://docs.godotengine.org/en/3.1/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) your branch as well -- it will be requested sooner or later.